packages/packsquash/src/vfs/os_fs.rs

```diff
@@ -96,23 +96,24 @@ fn is_system_or_hidden_file(entry: &DirEntry) -> bool {
 	}
 
 	// List based on https://www.toptal.com/developers/gitignore/api/git,windows,linux,macos
-	file_name.starts_with(b".")
-		|| if entry.file_type().is_file() {
-			file_name == b"desktop.ini"
-				|| file_name == b"Desktop.ini"
-				|| file_name == b"Thumbs.db"
-				|| file_name == b"ehthumbs.db"
-				|| file_name == b"ehthumbs_vista.db"
-				|| file_name.ends_with(b".lnk")
-				|| file_name.ends_with(b".orig")
-				|| file_name.ends_with(b".bak")
-				|| file_name.ends_with(b".tmp")
-		} else {
-			file_name == b"Network Trash Folder"
-				|| file_name == b"Temporary Items"
-				|| file_name == b"$RECYCLE.BIN"
-				|| file_name == b"__MACOSX"
-		}
+	if entry.file_type().is_file() {
+		(file_name.starts_with(b".") && !file_name.ends_with(b".mcfunction"))
```

Comment:
What do you think about tweaking the right-hand expression in this condition to do an inequality comparison like this? It's less characters overall and, to me at least, slightly easier to read.

```suggestion
		(file_name.starts_with(b".") && file_name != b".mcfunction")
```

Reply:
Changing it that way would likely exclude filenames like `.a.mcfunction`.

Ah... In that case, it might be cleaner to first check that the file doesn't have a specific extension. I'll consider making that change later.